x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -9362,11 +9362,9 @@
     "xpack.enterpriseSearch.appSearch.engine.curations.manageQueryButtonLabel": "クエリを管理",
     "xpack.enterpriseSearch.appSearch.engine.curations.manageQueryDescription": "このキュレーションのクエリを編集、追加、削除します。",
     "xpack.enterpriseSearch.appSearch.engine.curations.manageQueryTitle": "クエリを管理",
-    "xpack.enterpriseSearch.appSearch.engine.curations.organicDocuments.emptyDescription": "表示するオーガニック結果はありません。上記のアクティブなクエリを追加または変更します。",
```

Comment:
I'm pretty sure you don't have to prune these old labels, and that they will be pruned automatically by the i18n process. This is something @constancecchen mentioned before (if I recall correctly), though I'm not sure I'd be able to find the PR where she talks about that or any documentation to back that up.

Reply:
I got CI failures from these unpruned ones, I'll reach out to you next time I see that happen.

Reply:
Re: i18n failures - if you run `node scripts/i18n_check --fix` it'll handle removing stuff for you automatically without having to open the file!

More reading: https://github.com/elastic/kibana-team/blob/master/localization.md (their diagram is neat and worth expanding, + the info around how and when they manage scheduling for i18n)

Reply:
> Re: i18n failures - if you run node scripts/i18n_check --fix it'll handle removing stuff for you automatically without having to open the file!

👍 I used that to remove these, just highlighting that they're not pruned automatically.